web/src/api/typings.ts

```diff
@@ -69,6 +69,12 @@ export type Dataset = {
     name: string;
 };
 
+export type Revision = {
+    revision: string;
+    date: string;
+    id: string | number;
```

Comment:
Could you elaborate on `id`? is it the same as `revision` field?

Reply:
yes it is, will change to string only